sphinx/docserver.py

```diff
@@ -3,13 +3,17 @@
 import threading
 import time
 import webbrowser
+import asyncio
 
 import flask
 import tornado
 from tornado.httpserver import HTTPServer
 from tornado.ioloop import IOLoop
 from tornado.wsgi import WSGIContainer
 
+# asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
+# needs to be uncommented for a Windows 10 + Python 3.8 config
```

Comment:
@nimishbongale I think we should just check platform and python version and do this automatically

Reply:
@bryevdv Okay, I agree.

Will make the necessary changes!